### DISCUSSION TITLE
Auto start application after log in

### DISCUSSION THREAD

**Discussion by Valiumlord** [ltsp/ltsp#734] [Q&A] [answered]
Hello, Im having trouble with getting Chrome to auto start after the client logs in. I made bash script file and linked it with ltsp.conf crontab. Maybe im doing something wrong, cant get my head around it.

ltsp.conf
```
[clients]
SEARCH_DOMAIN="xxxx.xxxx xxxx.xxxxx.xx"
AUTOLOGIN=test1
RELOGIN=1
PASSWORDS_test1="xxxx/xxxxxxx"
CRONTAB_test1=@reboot /usr/bin/chrome-test.sh
```
chrome-test.sh
```
#!/bin/bash

google-chrome
```

**Comment by alkisg** [comment, marked as answer]
To autostart programs inside the user session, use /etc/xdg/autostart, not cron.

**Reply by Valiumlord** [reply]
Thank you, it worked. 
I had to "ltsp image /" to make it work
